Makefile

```diff
@@ -747,6 +750,11 @@ npm-cache:
 	touch -c node_modules
 
 node_modules:
+ifneq ($(NODEJS),)
+ifneq ($(NODE_VERSION),$(shell $(NODEJS) --version | sed 's/\(v[0-9]*\.[0-9]*\)\.[0-9]*/\1/'))
```

Comment:
(FS stands for Field Separator -- we output $1 and $2 with the field separator we use to split)